src/modules/data/entities/aggregate/common/character.embed.ts

```diff
@@ -36,12 +36,18 @@ export default class CharacterEmbed {
     })
     battleRank: number;
 
-    @ApiProperty({example: true, description: 'Character is ASPed'})
+    @ApiProperty({example: 1, description: 'Character ASP Rank'})
     @Column({
-        type: 'boolean',
-        default: false,
+        type: 'number',
+        default: 0,
+    })
+    asp: number = 0;
```

Comment:
This type is inferred from L41, so just remove `number` please :) Linter then should be happy, not that any of them pass in other areas :P I really need to fix them